### ISSUE TITLE
Refactor cost

### ISSUE THREAD

**Issue by baderbuddy** [james-atkinson/speedcomplainer#14]
In the config it would be helpful to refactor the $100 in the messages into a separate config variable.

**Comment by virtadpt** [comment]
Agreed.  I hand-modified them but it would be much more useful if it was a separate variable (plus it would make it easier to use with other services, like Uverse).
